src/helm/benchmark/run_specs/bhasa_run_specs.py

```diff
@@ -0,0 +1,689 @@
+
+# 1.3 Tamil: IndicXNLI
+@run_spec_function("indicxnli")
+def get_indicxnli_spec() -> RunSpec:
+    name = "indicxnli"
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions="உங்களுக்கு இரண்டு வாக்கியங்கள், X மற்றும் Y, தரப்படும்.\nபின்வரும் கூற்றுகளில் எது X மற்றும் Y வாக்கியங்களுடன் மிகப் பொருந்துகிறது எனக் கண்டறியவும்.\nA: X உண்மை என்றால் Y உம் உண்மையாக இருக்க வேண்டும்.\nB: X உம் Y உம் முரண்படுகின்றன.\nC: X உண்மையாக இருக்கும்போது Y உண்மையாக இருக்கலாம் அல்லது இல்லாமல் இருக்கலாம்.\nA அல்லது B அல்லது C என்ற ஒறே எழுத்தில் மட்டும் பதிலளிக்கவும்.",
+        output_noun="பதில்",
+        stop_sequences=["\n"],
+        max_tokens=8,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.IndicXNLIScenario"
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_f1_metric_specs(),
+        groups=["bhasa_nlr"],
+    )
+
+# 2. Causal Reasoning: XCOPA
+XCOPA_PROMPTS = {
+    "id": {
+        "input_noun": "Situasi",
+        "output_noun": "Jawaban",
+    },
+    "ta": {
+        "input_noun": "சூழ்நிலை",
+        "output_noun": "பதில்",
+    },
+    "th": {
+        "input_noun": "สถานการณ์",
+        "output_noun": "คำตอบ",
+    },
+    "vi": {
+        "input_noun": "Tình huống",
+        "output_noun": "Câu trả lời",
+    }
+}
+
+def generate_xcopa_run_spec(language="id"):
+    name = f"xcopa_{language}"
+
+    adapter_spec = get_generation_adapter_spec(
+        input_noun=XCOPA_PROMPTS[language]['input_noun'],
+        output_noun=XCOPA_PROMPTS[language]['output_noun'],
+        stop_sequences=["\n"],
+        max_tokens=8,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.XCOPAScenario",
+        args={
+            "language": language,
+        },
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_f1_metric_specs(),
+        groups=["bhasa_nlr"],
+    )
+
+@run_spec_function("xcopa")
+def get_xcopa_spec(language="id") -> RunSpec:
+    return generate_xcopa_run_spec(language)
+
+# D. Linguistic Diagnostics
+#   1. Syntax: Minimal Pairs
+#   2. Semantics: Pragmatic Reasoning (single sentece)
+#   3. Semantics: Pragmatic Reasoning (sentence pair)
+
+# 1. Syntax: LINDSEA Minimal Pairs
+lindsea_syntax_minimal_pairs_prompts = {
+    "id": {
+        "output_prefix": "Jawablah dengan satu huruf A atau B saja.",
+        "output_noun": "Jawaban",
+    },
+}
+
+@run_spec_function("lindsea_syntax_minimal_pairs")
+def get_lindsea_syntax_minimal_pairs_spec(language="id", mcq=False, calibrated=False) -> RunSpec:
+    name = f"lindsea_syntax_minimal_pairs_{language}"
+
+    if mcq and calibrated:
+        adapter_spec = get_multiple_choice_separate_adapter_spec(
+            method=ADAPT_MULTIPLE_CHOICE_SEPARATE_CALIBRATED,
```

Comment:
Calibrated actually refers to normalizing by the probability of "Answer: {reference}", no need for this mode